### ISSUE TITLE
Keep schema always up-to-date in Schema Manager

### ISSUE THREAD

**Issue by jstablesdf** [dreamfactorysoftware/dsp-core#60]
Schema does not automatically stay in sync with the database. Example, if from PhpMyAdmin I add or delete or make changes to a column, the schema manager does not reflect this change until I click on Update Schema. If I am viewing the schema manager when I make a change to the database via PhpMyAdmin I understand the page not updating, but if I navigate to another menu item and then navigate back to schema manager or even refresh the browser, the schema manager still reflects the old schema and I have to click update schema to get the new changes. I am assuming this is some sort of caching mechanism.

Basically each time you navigate to Schema Manager or refresh the browser it should have the current database schema. It's better to reflect the real schema of the database versus worrying about caching and showing an outdated schema to an unsuspecting user.

_this was split from Issue #58_

**Comment by jstablesdf** [comment]
The Refresh button was added to fulfill this use case. Schema are not constantly auto-fetched, but can be forced to refresh using this button in the UI.

![image](https://cloud.githubusercontent.com/assets/9806362/6046496/be1b9ff6-ac70-11e4-9815-49d1644856a8.png)
